### PR TITLE
set up postgres, fixed some css

### PR DESCRIPTION
**Pull request by mikesholiu**
*the name of this branch is misleading, didn't finish facebook auth yet

### CODE REVIEW

**Comment by chug2k** [comment]
Be careful - don't check in your secrets. Remember figaro and all that? There are a few ways to handle this but right now your send grid password and Facebook secret are available to the world.

**Comment by mikesholiu** [comment]
does this work? i added secret yml to the gitignore file. not sure if this covers all bases though because i thought the application.yml was already in there??

File: app/views/welcome/index.html.erb [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
-<div class="row marketing">
-  <div class="col-md-3"></div>
-  <div class="col-md-3" align="center">
-    <h4><b>Create public wikis.</b></h4>
-    <img src="test-tube-icon.svg" height="100" width="100">
+<div class="welcome-container">
+  <div class="jumbotron">
+    <p class="lead">Create and share markdown wikis</p>
+    
+    <p align="center"><%= link_to "Sign up today", new_user_registration_path, class: "btn btn-success" %></p>
```
Comment (chug2k) [inline app/views/welcome/index.html.erb]: remove `align=center`

**Comment by chug2k** [comment]
one tiny note, and feel free to merge! thanks for using the PR system. =)

**Comment by mikesholiu** [comment]
made a bunch of other changes, let me know if ok to merge (the branch should be called auth with twitter)

**Pull request merged**